samwise.js

```diff
@@ -1,46 +1,46 @@
 /**
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.